tests/fixtures.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+import graphene
+import flask
+
+from flasql.views import GraphQLView
+
+
+class Query(graphene.ObjectType):
+    debug = graphene.String()
+
+    def resolve_debug(self, args, context, info):
+        return 'Hi from GraphQL'
+
+
+def create_graphql_view(**kwargs):
+    schema = kwargs.pop('schema')
+
+    kwargs.update({
+        'schema': schema,
+        'enable_graphiql': True,
+    })
+
+    return GraphQLView.as_view('graphql', **kwargs)
+
+
+def create_app(**kwargs):
+    app = flask.Flask(__name__)
+    app.testing = True
+
+    app.add_url_rule('/graphql', view_func=create_graphql_view(**kwargs))
+
+    return app
+
+
+def create_schema():
+    return graphene.Schema(
+        query=Query,
+    )
+
+
+@pytest.fixture()
+def app():
+    """
+    Supplies a basic Flask app with a graphene schema
+    mounted inside the flasql view at /graphql.
+
+    """
+    app = create_app(schema=create_schema())
+    with app.test_request_context():
+        yield app
```

Comment:
@jimrollenhagen when you inject `app` into a test you're implicitly opening a new request context via flask so that it's available inside of the test. This is why I added the empty flask app. Ideally though that flask app will eventually get used to do some real world http->graphene testing.